install_config/aggregate_logging.adoc

```diff
@@ -655,8 +655,7 @@ specified, it is created with a request for that size.
 
 [WARNING]
 ====
-Using NFS storage as a volume or a persistent volume (or via NAS such as
-Gluster) is not supported for Elasticsearch storage, as Lucene relies on file
+Using NFS storage as a volume or a persistent volume is not supported for Elasticsearch storage, as Lucene relies on file
```

Comment:
Is there a reason these two lines were merged? Seems inconsistent with the rest of the formatting.